src/Controls/tests/UITests/Tests/DragAndDropUITests.cs

```diff
@@ -51,6 +51,8 @@ public void DragAndDropBetweenLayouts()
 			App.EnterText("TargetView", "DragAndDropBetweenLayouts");
 			App.Tap("GoButton");
 
+			App.Tap("ResetButton");
```

Comment:
I wonder if we need to "wait" that Reset finished

Reply:
After the reset we wait for the elements to show on screen, so I think we should be alright.